src/scripts.js

```diff
@@ -27,6 +33,49 @@ const handleData = (data) => {
   currentUser.sleep = new Sleep(data[1].sleepData, currentUser.id);
   currentUser.activity = new Activity(data[2].activityData, currentUser);
   updateDOM(currentUser, users);
+
+  const sendData = (e) => {
+    e.preventDefault();
+    let formData = new FormData(e.target);
+
+    if (e.target.name === 'activity') {
+      const newActivity = {
+        userID: currentUser.id,
+        date: formData.get('date').split('-').join('/'),
+        numSteps: formData.get('numSteps'),
+        minutesActive: formData.get('minutesActive'),
+        flightsOfStairs: formData.get('flightsOfStairs')
+      }
+    postData(e.target.name, newActivity);
+    }
+
+    if (e.target.name === 'hydration') {
+      const newHydrate = {
+        userID: currentUser.id,
+        date: formData.get('date').split('-').join('/'),
+        numOunces: formData.get('numOunces')
+      }
+
+    postData(e.target.name, newHydrate);
+    }
+
+    if (e.target.name === 'sleep') {
+      const newSleep = {
+        userID: currentUser.id,
+        date: formData.get('date').split('-').join('/'),
+        hoursSlept: formData.get('hoursSlept'),
+        sleepQuality: formData.get('sleepQuality')
+      }
+
+    postData(e.target.name, newSleep);
```

Comment:
I think we could at least refactor these postData function calls into a ternary operator at the end? This is definitely a tough one to refactor, though.